onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py

```diff
@@ -0,0 +1,100 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ....proto import onnx_proto
+from ...common._registration import register_converter
+
+
+def convert_sklearn_naive_bayes(scope, operator, container):
+    # Multinomial NB
+    # y = argmax (class_log_prior + X . feature_log_prob^T)
+    # Bernoulli NB
+    # y = argmax (class_log_prior + Σ neg_prob - X . neg_prob)
+    # neg_prob = log( 1 - e ^ feature_log_prob)
+
+    nb = operator.raw_operator
+    class_log_prior = nb.class_log_prior_.astype('float32')
+    feature_log_prob = nb.feature_log_prob_.T.astype('float32')
+    shape = [1, -1]
+    output_shape = [-1,]
+
+    class_log_prior_name = scope.get_unique_variable_name('class_log_prior')
+    feature_log_prob_name = scope.get_unique_variable_name('feature_log_prob')
+    sum_result_name = scope.get_unique_variable_name('sum_result')
+    cast_result_name = scope.get_unique_variable_name('cast_result')
+    reshaped_result_name = scope.get_unique_variable_name('reshaped_result')
+    shape_name = scope.get_unique_variable_name('shape')
+    output_shape_name = scope.get_unique_variable_name('output_shape')
+    argmax_output_name = scope.get_unique_variable_name('argmax_output')
+    cast2_result_name = scope.get_unique_variable_name('cast_result2')
+    reshaped2_result_name = scope.get_unique_variable_name('reshaped2_result')
+
+    container.add_initializer(shape_name, onnx_proto.TensorProto.INT64,
+                              [len(shape)], shape)
+    container.add_initializer(output_shape_name, onnx_proto.TensorProto.INT64,
+                              [len(output_shape)], output_shape)
+    container.add_initializer(class_log_prior_name, onnx_proto.TensorProto.FLOAT,
+                              class_log_prior.shape, class_log_prior)
+    container.add_initializer(feature_log_prob_name, onnx_proto.TensorProto.FLOAT,
+                              feature_log_prob.shape, feature_log_prob.flatten())
+
+    container.add_node('Reshape', [class_log_prior_name, shape_name], 
```

Comment:
class_log_prior_name and shape_name are names of static variables. Can we just compute the result here instead of asking runtime to do the Reshape?